### PR TITLE
Remove unused events

### PR DESCRIPTION
**Pull request by bilgili**
Removed events are not referenced by other projects

### CODE REVIEW

File: zeroeq/hbp/CMakeLists.txt [hunk 1]
```diff
@@ -3,11 +3,7 @@
 # All rights reserved. Do not distribute without further notice.
 
 flatbuffers_generate_c_headers(HBP_FBS
-  detail/camera.fbs
```
Comment (hernando) [inline zeroeq/hbp/CMakeLists.txt]: Keep this for the moment. It's needed by Neurscheme and I'm not removing it yet from RTNeuron.

Comment (bilgili) [inline zeroeq/hbp/CMakeLists.txt]: Yep, I will, I will also keep heartbeat

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by jafyvilla** [comment]
+1

**Pull request merged**